changelog.d/9749.misc

```diff
@@ -0,0 +1 @@
+Run `pyupgrade --py36-plus` over the codebase.
```

Comment:
This'll be the newsfile that i will use across all `pyupgrade` PRs, any adjustments?